ogr/ogrsf_frmts/pg/ogr_pg.h

```diff
@@ -600,6 +600,8 @@ class OGRPGDataSource final : public OGRDataSource
     int bHasLoadTables = false;
     CPLString osActiveSchema{};
     int bListAllTables = false;
+    bool m_bSkipViews = false;
```

Comment:
It appeared to me that newer flags use `bool` with an `m_` prefix so this is what I did. If there's some reason `int` should be used, let me know.